Automattic/wp-calypso
# Check platform before showing migration modal

We should check the imported site platform before showing the migration modal.
The migration modal will eventually lead to a new support ticket being created and our support team cannot assist with customer migrations if the platform of the site they want to import is not WordPress.

Related to #

## Proposed Changes

* Check platform and only show modal if it is wordpress

## Why are these changes being made?


* To ensure that no migration tickets are created when the customer's site is not on WordPress.

## Testing Instructions
- Using `/start` select that your goal is to migrate existing site.
- Enter non WordPress site
- Select on the next step WordPress
- When arriving on the plans page, click the back button - **The migration modal should not show**
- Regression Test: Follow the same flow with a WordPress site and verify that the modal is shown.






## Pre-merge Checklist
